roundabout-api/src/main/java/at/fhv/itm3/s2/roundabout/api/entity/IRoundaboutStructure.java

```diff
@@ -2,21 +2,72 @@
 
 import desmoj.core.simulator.Model;
 
+import java.util.Map;
 import java.util.Set;
 
 public interface IRoundaboutStructure {
 
+    /**
+     * Add a street connector to the structure.
+     *
```

Comment:
maybe a short info for the user, that the streetconnector is only added and the link to another streetconnection is not done here